_pages/policies/travel-and-leave-policies/travel-guide-faq.md

```diff
@@ -162,18 +162,18 @@ On the **Trip Booking Information** page you must enter some general information
 
 **Trip name:** Identify the budget of your trip here, based on the type of project you are traveling for:
 
-_Billable projects:_ for 18F and CoE, include the exact name and number of the project as it appears in [Tock](https://tock.18f.gov/projects). For PIF billable travel, simply indicate the project is billable
-_All other projects:_ include the name of the team budget that is paying for this travel (a list of options is in cell C5 of the [TTS Budget and Accounting Lookup](https://docs.google.com/spreadsheets/d/1twEX5wrriQ3Tbn25wN4n8rZPF9h5NqRQWIskkW6xQpY/edit#gid=0)
+* _Billable projects:_ for 18F and CoE, include the exact name and number of the project as it appears in [Tock](https://tock.18f.gov/projects). For PIF billable travel, simply indicate the project is billable.
+* _All other projects:_ include the name of the team budget that is paying for this travel (a list of options is in cell C5 of the [TTS Budget and Accounting Lookup](https://docs.google.com/spreadsheets/d/1twEX5wrriQ3Tbn25wN4n8rZPF9h5NqRQWIskkW6xQpY/edit#gid=0)
 
 ***Note:** If you are traveling for multiple projects (whether billable or not), list all of the relevant Tock name and numbers and budget names as appropriate.
 
 Examples:
```

Comment:
I think this list formatting isn't parsing correctly — try adding a line break after `Examples:` and I think it should work.

Reply:
Looks good to me-- do I need to go in and make these changes or are they already good to go once I hit "resolve conversation"?

Reply:
No, you'll need to go in and make that change — I tried, but was unable to edit the branch for some reason.